app/javascript/src/components/GameLogEntry.vue

```diff
@@ -1,17 +1,132 @@
 <template>
-  <div class="border border-black rounded p-2 m-2">
+  <div v-if="Array.isArray(event)" class="border border-black rounded p-2 m-2 rondel">
+    <div v-for="(action, index) in event" :key="(index + Math.random()).toString()">
+      <div v-if="action.type === 'rondel'">
+        <div>
+          <svg 
+            class="inline-block mr-1" 
+            xmlns="http://www.w3.org/2000/svg" 
+            width="30" 
+            height="20"
+            >
+            <Flag :nation="action.payload.nation.value" :width="(30).toString()"></Flag>
```

Comment:
I know this is a (bad) pattern I have elsewhere in the codebase but you can do:
```html
width="30"
```
Without the `:`, you don't need to `toString()` the 30.

Reply:
Ok, I still need to send it as a string so I'm changing it to `:width="'30'"`